apps/meteor/app/lib/server/functions/deleteMessage.ts

```diff
@@ -43,6 +43,7 @@ export async function deleteMessage(message: IMessage, user: IUser): Promise<voi
 
 	if (deletedMsg?.tmid) {
 		await Messages.decreaseReplyCountById(deletedMsg.tmid, -1);
+		await ReadReceipts.decrementThreadMessagesCountById(deletedMsg.tmid);
```

Comment:
I'm curious, why we store this info in the readreceipts model?

Reply:
I first thought of keeping the counter in order to erase data from the new `message_reads` collection when the counter hits zero... (not in this PR, but we could do that at some point thinking )
but I'm not sure if we'd like to erase data from that collection, since once a message turns into a thread, we won't ever turn it into a regular message once again. Anyways, I updated this to a flag now